### PR TITLE
Fix syntax for 'file is checkedout' warning banner and removal of hyphens

### PR DESCRIPTION
**Pull request by hmoco**

### CODE REVIEW

File: website/static/js/filepage/index.js [hunk 1]
```diff
@@ -106,10 +106,10 @@ var FileViewPage = {
                 self.file.checkoutUser = resp.data.relationships.checkout.links.related.href ? ((resp.data.relationships.checkout.links.related.href).split('users/')[1]).replace('/', ''): null;
                 if ((self.file.checkoutUser) && (self.file.checkoutUser !== self.context.currentUser.id)) {
                     m.render(document.getElementById('alertBar'), m('.alert.alert-warning[role="alert"]', m('span', [
-                        m('strong', 'File is checked-out.'),
-                        'This file has been checked-out by a ',
+                        m('strong', 'File is checked out.'),
+                        'nThis file has been checked out by a ',
```
Comment (jinluyuan) [inline website/static/js/filepage/index.js]: "n" should be space?

File: website/static/js/filepage/index.js [hunk 2]
```diff
@@ -260,7 +260,7 @@ var FileViewPage = {
                     }).done(function(resp) {
                         window.location.reload();
                     }).fail(function(resp) {
-                        $osf.growl('Error', 'Unable to force check-in file, make sure you have admin privileges.');
+                        $osf.growl('Error', 'Unable to force check in file, make sure you have admin privileges.');
```
Comment (jinluyuan) [inline website/static/js/filepage/index.js]: Change comma after "file" to period, and capitalize "Make".

File: website/static/js/filepage/index.js [hunk 3]
```diff
@@ -106,10 +106,10 @@ var FileViewPage = {
                 self.file.checkoutUser = resp.data.relationships.checkout.links.related.href ? ((resp.data.relationships.checkout.links.related.href).split('users/')[1]).replace('/', ''): null;
                 if ((self.file.checkoutUser) && (self.file.checkoutUser !== self.context.currentUser.id)) {
                     m.render(document.getElementById('alertBar'), m('.alert.alert-warning[role="alert"]', m('span', [
-                        m('strong', 'File is checked-out.'),
-                        'This file has been checked-out by a ',
+                        m('strong', 'File is checked out.'),
+                        'nThis file has been checked out by a ',
                         m('a[href="/' + self.file.checkoutUser + '"]', 'collaborator'),
-                        '. It beeds to be checked back in before any changes can be made.'
+                        '. It needs to be checked back in before any changes can be made.'
```
Comment (jinluyuan) [inline website/static/js/filepage/index.js]: Take out the word "back"

File: website/static/js/filepage/index.js (hunk 1 above)
Comment (sloria) [inline website/static/js/filepage/index.js]: +1

File: website/static/js/filepage/index.js (hunk 3 above)
Comment (sloria) [inline website/static/js/filepage/index.js]: +0

File: website/static/js/filepage/index.js (hunk 2 above)
Comment (sloria) [inline website/static/js/filepage/index.js]: +1

**Comment by sloria** [comment]
Changes made per Jinlu's suggestions. Merged into release/0.55.0

**Pull request closed without merging**